src/Sylius/Behat/Context/Api/Shop/ProductVariantContext.php

```diff
@@ -137,14 +137,15 @@ public function iShouldSeeVariantIsDiscountedFromToWithOnlyPromotion(
         int $price,
         string $promotionName
     ): void {
-        $content = $this->findVariant($variant);
-
-        Assert::same(sizeof($content['appliedPromotions']), 1);
-        Assert::same($content['price'], $price);
-        Assert::same($content['originalPrice'], $originalPrice);
-        $catalogPromotionCode = StringInflector::nameToCode($promotionName);
-        Assert::same($content['appliedPromotions'][$catalogPromotionCode]['translations']['en_US']['name'], $promotionName);
-        Assert::same($content['appliedPromotions'][$catalogPromotionCode]['translations']['en_US']['description'], $promotionName . ' description');    }
+        $productVariant = $this->findVariant($variant);
```

Comment:
```suggestion
        $variantContent = $this->findVariant($variant);
```
or sth similar, because `$productVariant` is misleading as it is not an entity